test/test_jit.py

```diff
@@ -6352,7 +6352,9 @@ class Config:
         premise = torch.LongTensor(48, 128).random_(0, 100)
         hypothesis = torch.LongTensor(24, 128).random_(0, 100)
 
-        self.checkTrace(SNLIClassifier(Config()), (premise, hypothesis), inputs_require_grads=False)
+        # TODO: toggle export_import once LSTM isn't a python op
```

Comment:
It should not be a python op right now, is this still the case? (changes landed a few days ago)